util/checks/index.ts

```diff
@@ -93,8 +179,16 @@ export type IssueGuidanceUrls = {
   databaseTableUrl: string;
   serverLogInsightsUrl: string;
   serverSchemaUrl: string;
+  sectionUrl: (mainUrl: string | undefined, section: string) => string | undefined;
 };
 
+export function sectionUrl (mainUrl: string | undefined, section: string): string | undefined {
```

Comment:
I wonder if this should maybe be called "featureUrl" instead? (since it points to a particular feature in-app, and to a page describing that feature in the public docs)

Reply:
Sure. I considered that but thought it was toss-up. I'll change to 'featureUrl'.